frontend/src/pages/TrainingReports/components/EventCard.js

```diff
@@ -25,16 +33,29 @@ function EventCard({
   const menuItems = [];
 
   if (event.status !== 'Complete') {
+    if (hasEditPermissions) {
+    // Create session.
+      menuItems.push({
+        label: 'Create session',
+        onClick: () => {
+          history.push(`/training-report/${event.id}/session/new/`);
+        },
+      });
+
+      // Edit event.
+      menuItems.push({
+        label: 'Edit event',
+        onClick: () => {
+          history.push(`/training-report/${event.id}/event-summary`);
+        },
+      });
+    }
+
+    // View event.
     menuItems.push({
-      label: 'Create session',
-      onClick: () => {
-        history.push(`/training-report/${event.id}/session/new/`);
-      },
-    });
-    menuItems.push({

```

Comment:
You should be able to view completed events

Reply:
Ok so should always add the view item?

Reply:
Yeah looks like it, might be worth reviewing the mockups to double check my thinking

Reply:
This is updated.